src/Widgets/Proxy/ProxyConfigurationPage.vala

```diff
@@ -34,9 +34,10 @@ namespace Network.Widgets {
         private Gtk.SpinButton ftp_spin;
         private Gtk.SpinButton socks_spin;
 
-        private Gtk.Button apply_button;
+        private Gtk.Switch *enable_sw;
 
-        public ConfigurationPage () {
+        public ConfigurationPage (Gtk.Switch *EnSwitch) {
```

Comment:
The same goes here. Also, don't use CamelCase for the name of the variable but instead something like `_root_switch`.